public/dorequest.php

```diff
@@ -232,11 +233,20 @@ function DoRequestError(string $error, ?int $status = 200, ?string $code = null)
         $response['Response'] = getGamesListDataNamesOnly($consoleID);
         break;
 
-    case "officialgameslist":
+    case "officialgameslist": // TODO: is this used anymore? It's not used by the DLL.
```

Comment:
It's (unfortunately) used by 3rd party front-ends tapping into the Connect API, eg: Batocera.

Reply:
Maybe we can reimplement in the web API somehow and have this fall through to the web API to avoid a breaking change?

Reply:
I would think that `API_GetGameList?f=1` would be preferred. It returns the ID/Title for each game of a console, along with the image, and numbers of achievements, leaderboards, and points.

Reply:
For reference: https://github.com/batocera-linux/batocera-emulationstation/blob/3959d0742f1b3fb9e3b86e56f365d5eea99eaf5d/es-app/src/RetroAchievements.cpp#L449

Reply:
Even better. `API_GetGameList?f=1&h=1` returns a list of ID/Titles for each game in a console, plus the known associated hashes. 

The only thing that would have to change is they'd have to query each console individually, or we'd have to open up `API_GetGameList` to support the "0" (all) console.

Reply:
I can't help wonder how useful this functionality even is for them, given how few of our consoles do full file hashes.